curriculum/challenges/english/03-front-end-development-libraries/redux/use-const-for-action-types.md

```diff
@@ -87,12 +87,8 @@ The `authReducer` function should handle multiple action types with a switch sta
 
 ```js
 const noWhiteSpace = __helpers.removeWhiteSpace(code);
-assert(
-  (/constLOGIN=(['"`])LOGIN\1/.test(noWhiteSpace) &&
-    /constLOGOUT=(['"`])LOGOUT\1/.test(noWhiteSpace)) ||
-      /const(LOGIN|LOGOUT)=(['"`])\1\2,(?!\1)(LOGIN|LOGOUT)=(['"`])\3\4/.test(noWhiteSpace) ||
-        /const\[(LOGIN|LOGOUT),(?!\1)(LOGIN|LOGOUT)\]=\[(['"`])\1\3,\3\2\3\]/.test(noWhiteSpace)
-);
+assert(LOGIN === 'LOGIN' && LOGOUT === 'LOGOUT')
+assert(noWhiteSpace.includes('const'))
```

Comment:
This will always be true, because there are many variables in the seed declared with `const`.